### ISSUE TITLE
audio elements must have a track for captions - does this rule apply to prerecorded audio-only media

### ISSUE THREAD

**Issue by melaniephilipp** [dequelabs/axe-core#686]
Does the rule "audio elements must have a track for captions" apply to prerecorded audio-only content: (https://dequeuniversity.com/rules/axe/2.6/audio-caption)? 

If yes: this is not a WCAG requirement. Per SC 1.2.1, audio-only content is required to have an "alternative for time-based media" where "alternative for time-based media" is a "document including correctly sequenced text descriptions of time-based visual and auditory information and providing a means for achieving the outcomes of any time-based interaction". So a transcript satisfies SC 1.2.1. SC 1.2.2 (referenced in https://dequeuniversity.com/rules/axe/2.6/audio-caption) is not applicable to prerecorded audio-only.

**Comment by WilcoFiers** [comment]
Hey Melanie. Thanks for your contribution. The audio-caption rule doesn't trigger violations. If it finds audio elements without a caption it'll put it in the Needs review category. We flag it as something to look at, but we're aware there are alternative ways to meet the requirement. The audio may even just be background, in which case nothing is needed.

Let me know if this answers your concern. If not, what suggestion would you have for improving the rule?

**Comment by melaniephilipp** [comment]
Hi Wilco, Thanks for the reply.

Here is what has me confused. The page: https://dequeuniversity.com/rules/axe/2.6/audio-caption lists the rule as a MUST for 1.2.2 with an impact of CRITICAL. If this is for audio-only, this should not be flagged under 1.2.2 at all, since that SC is for "prerecorded audio content in synchronized media" (aka video with audio).

Raising a flag for a reviewer to check that audio-only content has an "alternative for time-based media" (typically a transcript, since most web audio players can't even provide timed captions, and transcripts also help people who are deafblind) per SC 1.2.1 makes perfect sense. And, if no "alternative for time-based media" exists for audio-only content that needs it, then that would logically be a critical issue.

To summarize:
- If this rule is triggered for audio-only content, it should reference SC 1.2.1 instead of SC 1.2.2, and the corresponding page info at https://dequeuniversity.com/rules/axe/2.6/audio-caption needs to change.
- Flagging it under Needs Review is great.
- If this rule is triggered by BOTH audio-only and audio-with-video, then the rule should be separated into two rules: one for 1.2.1 and one for 1.2.2.

Thanks for listening, Wilco. If you'd like to take this discussion offline, you know how to find me :-)

**Comment by WilcoFiers** [comment]
I think you're correct. The audio-captions rule should reference 1.2.1 and not 1.2.2. That's an easy fix. Thanks for reporting.

**Comment by anutin544** [comment]
Ok

**Comment by SebacicA11y** [comment]
hey @WilcoFiers, I'm trying to get my head around what I see as a discrepancy between https://dequeuniversity.com/rules/axe/3.2/audio-caption?application=msftAI and 
https://www.w3.org/WAI/WCAG21/Techniques/general/G158.html

Deque Uni says "<audio> must have a captions <track>" but WCAG general 158 suggests a transcript on the website is sufficient. 

What am I missing?
